params_test.go

```diff
@@ -231,14 +232,6 @@ func TestParseJSONBodyMux(t *testing.T) {
 	if val != true {
 		t.Fatal("Value of 'test' should be 'true', got: ", val)
 	}
-
-	val, present = params.Get("id")
-	if !present {
-		t.Fatal("Key: 'id' not found")
-	}
-	if val != uint64(42) {
-		t.Fatal("Value of 'id' should be 42, got: ", val)
-	}
```

Comment:
correct, gorilla mux was responsible for parsing url params and passing them through the context it created